jaeger-core/src/test/java/io/jaegertracing/JaegerSpanTest.java

```diff
@@ -282,6 +313,32 @@ public void testLog() {
     assertEquals(expectedFields, actualLogData.getFields());
   }
 
+  @Test
+  public void testLogAfterFinish() {
+    final long expectedTimestamp = 2222;
+    final String expectedLog = "some-log";
+    final String expectedEvent = "expectedEvent";
+
+    when(clock.currentTimeMicros()).thenReturn(expectedTimestamp);
+
+    jaegerSpan.finish();
+    jaegerSpan.log(expectedEvent);
+    assertNull(jaegerSpan.getLogs());
+  }
+
+  @Test
+  public void testLogWithTimestampAfterFinish() {
+    Map<String, String> unexpectedFields = new HashMap<String, String>() {
```

Comment:
It's cleaner to use [`Collections.singletonMap("foo", "bar")`](https://docs.oracle.com/javase/7/docs/api/java/util/Collections.html#singletonMap(K,%20V)) in this case.